src/main/java/uk/gov/pay/adminusers/model/Role.java

```diff
@@ -73,7 +71,7 @@ public boolean equals(Object o) {
             return false;
         }
 
-        return isEqualCollection(permissions, role.permissions);
+        return permissions == null && role.permissions == null || permissions.equals(role.permissions);
```

Comment:
This could actually be `return Objects.equals(permissions, role.permissions);`, which takes care of the nulls. However, it doesn’t look like either argument could actually be null: in fact, `isEqualCollection(…)` will throw a `NullPointerException` is either of its arguments are null, so we can just do `return permissions.equals(role.permissions);`.

However, using `List`’s `equals(…)` method would mean we’re relying on the order of the elements in the two lists being the same. Now they might well be but I’m not sure.

Probably the best approach would be to use a `Set` rather than a `List` to hold collections of permissions, since  it makes no sense to have a permission more than once. `Set`’s `equals(…)` method doesn’t care about order, which would allow us to `return permissions.equals(role.permissions);`. This doesn’t look like a huge change, so I’ll see about doing that today.